shell/android-studio/reicast/src/main/res/values/strings.xml

```diff
@@ -133,10 +133,6 @@
     <string name="downloadVMU">Download VMU</string>
     
     <string name="log_saved">Logcat saved and copied to clipboard\nPlease paste inside the Github issue</string>
-
-    <string name="service_ticker">reicast is running...</string>
-    <string name="service_title">reicast is running...</string>
-    <string name="service_content">Emulator service is currently running</string>
```

Comment:
Was the removing of those lines intentional?

Reply:
https://github.com/reicast/reicast-emulator/commit/a9ea64f36a293f28507977ef403e4e9540cdc59c

Reply:
Ok, so they weren't used anymore.